From 0b7076c253635e6743e755a22e2759bfe74b128e Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Wed, 4 Sep 2024 22:17:56 +0800
Subject: [PATCH 26/35] [r38451] vd_ffmpeg: Remove reference to slice_offset
 which was removed from FFmpeg

This field isn't used anymore in MPlayer since a long time and the
remaining av_freep call is probably just a left over.

Removed from lavc on bump to 61

(From upstream SVN r38451)
---
 libmpcodecs/vd_ffmpeg.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/libmpcodecs/vd_ffmpeg.c b/libmpcodecs/vd_ffmpeg.c
index 6e30c51..c22397a 100644
--- a/libmpcodecs/vd_ffmpeg.c
+++ b/libmpcodecs/vd_ffmpeg.c
@@ -526,7 +526,6 @@ static void uninit(sh_video_t *sh){
 
         av_freep(&avctx->extradata);
         av_freep(&avctx->hwaccel_context);
-        av_freep(&avctx->slice_offset);
     }
 
     avcodec_free_context(&avctx);
-- 
2.46.0

